source/braille.py

```diff
@@ -1698,6 +1713,11 @@ def handleUpdate(self, obj):
 				break
 		else:
 			# No region for this object.
+			# There are some objects that may require an update even if they have no region.

```

Comment:
Please change "require an update" to "require special update behavior" or similar. "Require an update" makes it look like we're updating the object itself, but in this case, we're updating another object because of this one.